plugins/aea-ledger-ethereum-hwi/setup.py

```diff
@@ -43,7 +43,7 @@
         "ipfshttpclient==0.8.0a2",
         "eth-account==0.5.6",
         "open-aea-ledger-ethereum~=1.31.0",
-        "apduboy>=0.5.0",
+        "apduboy @ git+https://github.com/LedgerHQ/apduboy.git",
```

Comment:
Shouldn't this be same as https://github.com/valory-xyz/open-aea/pull/614/files?file-filters%5B%5D=.ini&file-filters%5B%5D=.lock&file-filters%5B%5D=.py&file-filters%5B%5D=.toml&file-filters%5B%5D=.txt&file-filters%5B%5D=.yml&file-filters%5B%5D=No+extension&show-deleted-files=true&show-viewed-files=true#diff-230078d672f10d17463a8a6265cad825b790885898256a3365be90685caac58dR53